src/main/java/frc/robot/Robot.java

```diff
@@ -58,7 +58,7 @@
       new WPI_VictorSPX(RoboRIO.kPortMotorDriveBackLeft);
   private final WPI_VictorSPX m_motorDriveBackRight =
       new WPI_VictorSPX(RoboRIO.kPortMotorDriveBackRight);
-  Compressor compressor = new Compressor(0);
+  private final Compressor compressor = new Compressor(RoboRIO.kPortCompressor);
```

Comment:
Should the `compressor` variable be defined as a member variable via the `m_` prefix, or no?